### PR TITLE
=doc #16399 Document async appender for SLF4J

### PR DESCRIPTION
**Pull request by johanandren**
Just did the Java one for feedback, will copy it to the Scala docs before this PR being done.

### CODE REVIEW

File: akka-docs/rst/java/logging.rst [hunk 1]
```diff
@@ -323,6 +328,18 @@ the first case and ``LoggerFactory.getLogger(String s)`` in the second).
...
+Using the SLF4J API directly
+----------------------------
+If you use the SLF4J API directly in your application, remember that log operations will block
+while logging implementation behind it writing the log entries to disk.
+
+This can be avoided by configuring the logging implementation to use
+a non-blocking appender. Logback provides `AsyncAppender <http://logback.qos.ch/manual/appenders.html#AsyncAppender>`_
+that does this. It also contains a feature which will drop ``INFO`` and ``DEBUG`` messages if the logging
+load is high.
+
```
Comment (patriknw) [inline akka-docs/rst/java/logging.rst]: I guess there is a corresponding scala rst file

**Comment by patriknw** [comment]
LGTM after adding same to scala

File: akka-docs/rst/scala/logging.rst [hunk 2]
```diff
@@ -359,6 +364,18 @@ the first case and ``LoggerFactory.getLogger(s: String)`` in the second).
 
   val log = Logging(system.eventStream, "my.nice.string")
 
+.. _slf4j-directly-scala:
+
+Using the SLF4J API directly
+----------------------------
+If you use the SLF4J API directly in your application, remember that log operations will block
+while logging implementation behind it writing the log entries to disk.
```
Comment (ktoso) [inline akka-docs/rst/scala/logging.rst]: grammar a bit weird here, let's

`... the logging operations will block while the underlying infrastructure writes the log statements.`

**Comment by ktoso** [comment]
LGTM, I'd adjust the wording a bit as I commented though

**Comment by ktoso** [comment]
Thanks, LGTM

**Comment by ktoso** [comment]
refs #16399

**Comment by ktoso** [comment]
Aaaa, we're not supposed to merge to master right now, right? Sorry, but it's only docs so nothing bad happened (uff).

**Comment by patriknw** [comment]
docs are allowed, no problem

**Pull request merged**